docs/source/basics.rst

```diff
@@ -176,6 +176,25 @@ There is more information about creating stubs in the
 The following sections explain the kinds of type annotations you can use
 in your programs and stub files.
 
+Mypy will also look in some default install directory to try to find annotations.
+For example, for UNIX based, with python 3.4, it will try to find it in
+- ``${HOME}/.local/shared/typehints/python3.4/``
+- ``/usr/shared/typehints/python3.4/``
```

Comment:
Can somebody summarize what the proposed PEP 484 change should be? I don't
think the shared path is set in stone, AFAIK nobody has actually
implemented it yet. And if there's an existing standard for how to name
files under ~/.local I'm all for it!